pages/mesosphere/dcos/version-policy/index.md

```diff
@@ -83,98 +80,97 @@ Customers running DC/OS on non-supported platform components should upgrade to a
-        <td><p style="text-align: center;">Docker CE 19.03.5<br>Docker CE 18.09.9</p></td>
     </tr>
 </table>
 
 ### <a name="Oracle-support-matrix"></a>Oracle support matrix
+*NOTE: Starting with DC/OS 2.0, we have discontinued support for Oracle Linux and have ceased qualifying it as an Operating System*.
 <table class="table">
     <tr>
     <th><strong>Platform Component</strong></th>
     <th><strong>DC/OS 1.13 Latest Stable</strong></th>
-    <th><strong>DC/OS 1.12 Latest Stable</strong></th>
     </tr>
     <tr>
         <td>Oracle Linux 7.6 </td>
         <td><p style="text-align: center;">Docker CE 18.09.2<br>Docker EE 18.09.2</p></td>
-        <td><p style="text-align: center;">Docker CE 18.09.2<br>Docker EE 18.09.2</p></td>
     </tr>
     <tr>
         <td>Oracle Linux 7.5 </td>
         <td><p style="text-align: center;">Docker CE 18.09.2<br>Docker EE 18.09.2</p></td>
-        <td><p style="text-align: center;">Docker CE 18.09.2<br>Docker EE 18.09.2</p></td>
     </tr>
     <tr>
         <td>Oracle Linux 7.4 </td>
         <td><p style="text-align: center;">Docker CE 18.09.2<br>Docker EE 18.09.2</p></td>
-        <td><p style="text-align: center;">Docker CE 18.09.2<br>Docker EE 18.09.2</p></td>
     </tr>
 </table>
 
 ### <a name="CoreOS-support-matrix"></a>CoreOS support matrix
-NOTE: Starting with DC/OS 2.0, we have discontinued support for CoreOS and have ceased qualifying it as an Operating System. This chart shows the final qualified versions of DC/OS for the final qualified versions CoreOS.
+As of May 26th 2020 CoreOS Container Linux has reached its [end of life](https://coreos.com/os/eol/) and will no longer receive updates.
+
+*NOTE: Starting with DC/OS 2.0, we have discontinued support for CoreOS and have ceased qualifying it as an Operating System*. This chart shows the final qualified versions of DC/OS for the final qualified versions CoreOS.
 <table class="table">
     <tr>
     <th><strong>Platform Component</strong></th>
     <th><strong>DC/OS 1.13 Latest Stable</strong></th>
-    <th><strong>DC/OS 1.12 Latest Stable</strong></th>
     </tr>
     <tr>
         <td>CoreOS 2079.3.0</td>
         <td><p style="text-align: center;">Docker CE 18.06.3</p></td>
-        <td><p style="text-align: center;">Docker CE 18.06.3</p></td>
     </tr>
     <tr>
         <td>CoreOS 2023.5.0</td>
         <td><p style="text-align: center;">Docker CE 18.06.1</p></td>
-        <td><p style="text-align: center;">Docker CE 18.06.1</p></td>
     </tr>
     <tr>
         <td>CoreOS 2023.4.0</td>
         <td><p style="text-align: center;">Docker CE 18.06.1</p></td>
-        <td><p style="text-align: center;">Docker CE 18.06.1</p></td>
+    </tr>
+</table>
+
+### <a name="flatcar-support-matrix"></a>Flatcar Linux support matrix
+<table class="table">
+    <tr>
+    <th><strong>Platform Component</strong></th>
+    <th><strong>DC/OS 2.1 Latest Stable</strong></th>
+    </tr>
+    <tr>
+        <td>Flatcar Linux 2345.3.1</td>
```

Comment:
Flatcar Linux has been qualified with the DC/OS 2.1.0-rc2 here: 
https://github.com/dcos/dcos-images/pull/536